package-scripts.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
         write: 'prettier --write \"./server/**/*.ts\"',
         check: 'prettier --list-different \"./server/**/*.ts\"'
       },
-      test: 'node ./tools/scripts/test ./dist/server/test',
+      test: 'node ./tools/scripts/test ./server/test',
```

Comment:
Oh sweet, does this mean we won't have to recompile to update server tests between runs?

Reply:
Yep that's correct!